mobi/src/main/java/com/mobiperf/MeasurementScheduler.java

```diff
@@ -81,6 +84,7 @@
 
   // This arbitrary id is private to Speedometer
   private static final int NOTIFICATION_ID = 1234;
+  private static final String CHANNEL_ID = "";
```

Comment:
how come the channel does not have an ID

Reply:
fixed